docs/operator-manual/feature-maturity.md

```diff
@@ -3,22 +3,83 @@
 Argo CD features may be marked with a certain [status](https://github.com/argoproj/argoproj/blob/main/community/feature-status.md)
 to indicate their stability and maturity. These are the statuses of non-stable features in Argo CD:
 
-| Feature                             | Introduced | Status |
-|-------------------------------------|------------|--------|
-| [Structured Merge Diff Strategy][1] | v2.5.0     | Beta   |
-| [AppSet Progressive Syncs][2]       | v2.6.0     | Alpha  |
-| [Proxy Extensions][3]               | v2.7.0     | Alpha  |
-| [Skip Application Reconcile][4]     | v2.7.0     | Alpha  |
-| [AppSets in any Namespace][5]       | v2.8.0     | Beta   |
-| [Dynamic Cluster Distribution][6]   | v2.9.0     | Alpha  |
-| [Server Side Diff][7]               | v2.10.0    | Beta   |
-| [Service Account Impersonation][8]  | v2.13.0    | Alpha  |
-
-[1]: ../user-guide/diff-strategies.md#structured-merge-diff
+!!! danger "Using Alpha/Beta features risks"
+
+    Aplha and Beta features do not guarantee backward compatibility and are subject to breaking changes in the future releases.
+    It is highly suggested for Argo users not to rely on these features in production environments, especially if you do not have
+    control over the Argo CD upgrades.
+
+    Furthermore, removal of Alpha features may modify your resources to an unpredictable state after Argo CD is upgraded.
+    You should make sure to document which features are in use and review the [release notes](./upgrading/overview.md) before upgrading.
+
+## Overview
+
+| Feature                                   | Introduced | Status |
+| ----------------------------------------- | ---------- | ------ |
+| [AppSet Progressive Syncs][2]             | v2.6.0     | Alpha  |
+| [Proxy Extensions][3]                     | v2.7.0     | Alpha  |
+| [Skip Application Reconcile][4]           | v2.7.0     | Alpha  |
+| [AppSets in any Namespace][5]             | v2.8.0     | Beta   |
+| [Cluster Sharding: round-robin][6]        | v2.8.0     | Alpha  |
+| [Dynamic Cluster Distribution][7]         | v2.9.0     | Alpha  |
+| [Server Side Diff][8]                     | v2.10.0    | Beta   |
+| [Cluster Sharding: consistent-hashing][9] | v2.12.0    | Alpha  |
+| [Service Account Impersonation][10]       | v2.13.0    | Alpha  |
+
 [2]: applicationset/Progressive-Syncs.md
 [3]: ../developer-guide/extensions/proxy-extensions.md
 [4]: ../user-guide/skip_reconcile.md
 [5]: applicationset/Appset-Any-Namespace.md
-[6]: dynamic-cluster-distribution.md
-[7]: ../user-guide/diff-strategies.md#server-side-diff
-[8]: app-sync-using-impersonation.md
+[6]: ./high_availability.md#argocd-application-controller
+[7]: dynamic-cluster-distribution.md
+[8]: ../user-guide/diff-strategies.md#server-side-diff
+[9]: ./high_availability.md#argocd-application-controller
+[10]: app-sync-using-impersonation.md
```

Comment:
Please always keep the link section at the bottom of the file for easy reference.

Reply:
The links section can be placed anywhere in the file. In this case, I think it is much easier to have these link directly under the main overview table.

Reply:
You are referencing the links in the new section below which makes it confusing to find where they are defined. This type of markdown links are called Reference and usually you put references at the bottom of the file. You can read more about it here:
https://google.github.io/styleguide/docguide/style.html#reference

Reply:
Let me explain in more details why I think, in this case, reference links should be put under the main table: if a contributor updates the page to add their new fields to the list, it is very easy to scroll down, add the links and be done with it. When that happens, the main overview table would become incomplete. If the links are placed directly underneath the main table, it becomes more obvious to update the overview table. 

But I'll move them to the bottom if markdown styling is more important.